app/javascript/application.js

```diff
@@ -0,0 +1,17 @@
+// Entry point for the build script in your package.json
+import '@hotwired/turbo-rails';
+import './controllers';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import App from './components/App';
+import store from './redux/store';
+
+/* eslint-disable react/no-deprecated */
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById('root'),
+);
+/* eslint-enable react/no-deprecated */
```

Comment:
- [ ] **[OPTIONAL]** I'm sorry the previous reviewer didn't point this out, but please avoid disabling linters as they are very useful for keeping a consistent code format and ensuring the use of best practices. Instead, please consider solving the errors! 🤓 🔧